Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 
 name = "spidev"
-version = "0.6.0"
+version = "0.6.2-alpha.0"
```

This looks wrong for the 0.6.1 release. Or am I somehow misunderstanding the release process?

```suggestion
version = "0.6.1"
```

---

Version 0.6.1 is set with [3e9b3b2](https://github.com/rust-embedded/rust-spidev/pull/50/commits/3e9b3b288588d3f7ec3c4dc2741a126bfd851273). This is the revision I planned for tagging release 0.6.1 and releasing to crates.io from.

For the time after 0.6.1 [ee8a1e5](https://github.com/rust-embedded/rust-spidev/pull/50/commits/ee8a1e566aecad4f3a5070e98881ec52fc21a581) attempts to cleanly state that the code in question is no longer 0.6.1 and there is less room for confusion.

---

Ah ok, that wasn't obvious when viewing the changes.

So this MR should be merged in "rebase and merge" mode, because "squash and merge" would destroy the intermediate commit, and the default mode "merge pull request" would create a slightly confusing commit history, right?